src/Benchmarks/SingleQueryEfMiddleware.cs

```diff
@@ -35,10 +35,8 @@ public SingleQueryEfMiddleware(RequestDelegate next)
             {
                 var db = (ApplicationDbContext)httpContext.RequestServices.GetService(typeof(ApplicationDbContext));
 
-                db.ChangeTracker.QueryTrackingBehavior = QueryTrackingBehavior.NoTracking;
-
                 var id = _random.Next(1, 10001);
-                var row = await db.World.FirstAsync(w => w.Id == id);
+                var row = await db.World.AsNoTracking().FirstAsync(w => w.Id == id);
```

Comment:
Would it be beneficial/allowed to cache the expression? We used codegen to do this in Razor because these can be deceptively expensive.